packages/ipfs-core/src/components/block/get.js

```diff
@@ -22,14 +22,14 @@ module.exports = ({ blockService, preload }) => {
    * console.log(block.data)
    * ```
    */
-  async function get (cid, options = {}) { // eslint-disable-line require-await
+  async function get (cid, options = {}) {
     cid = cleanCid(cid)
 
     if (options.preload !== false) {
       preload(cid)
     }
 
-    return blockService.get(cid, options)
+    return await blockService.get(cid, options)
```

Comment:
You are right it may not be relevant to the actual objective of this PR. However, I would still argue in favor of this change because:

1. I would much rather have `await` here than tell eslint to shut up about not having `await.`.
2. It is more clear to a reader (which may not be aware of this odd behavior of async functions) what is going on.

If this is a problem, I can change it back.

Reply:
I don't like doing the lint disable to make the function `async` as we do. We had a discussion about this during the async refactor (I could not really find it now...) and this was the way decided.

I don't really want that we take different approaches across the codebase. We can talk about it, perhaps through an issue mentioning the ipfs team.

Reply:
Totally agree! Should we open an issue mentioning the entire JS team to get this rolling?

Reply:
Created an issue for that https://github.com/ipfs/js-ipfs/issues/3392